### DISCUSSION TITLE
Problems with GitHub Education

### DISCUSSION THREAD

**Discussion by andiricum2** [community/community#66027] [GitHub Education]
### Hi everyone,

I applied to the github education program and they rejected me having sent the data correctly. At first I received an email saying that I did not have my name added to my github account and that I had to do it so they could verify my identity. So I put my name and reapplied and since then I have not received any response, this was about a month ago. In my opinion I think something has happened with my application, I need to have the github education for this week since I start classes on Thursday, thanks for helping me!

**Comment by arafajs** [comment]
In this case, I recommend that you contact GitHub support for assistance.
Go to the GitHub Support website: https://github.com/contact.
